pkg/registry/server/client.go

```diff
@@ -0,0 +1,118 @@
+package server
```

Comment:
`client.go` in `package server`? lol

Reply:
Honestly our package game is :hankey: :rofl: 

Any suggestions? I think I previously had `pkg/registry/client/client.go` but then decided to merge them together b/c the client is coupled with the server via param names etc.